frontend/client/utils/api.ts

```diff
@@ -145,6 +145,12 @@ export function massageSerializedState(state: AppState) {
       (state.proposal.detail.funded as any) as string,
       16,
     );
+    if (state.proposal.detail.rfp && state.proposal.detail.rfp.bounty) {
+      state.proposal.detail.rfp.bounty = new BN(
+        (state.proposal.detail.rfp.bounty as any) as string,
```

Comment:
Just curious, why override the BN type as string here?

Reply:
The TS type thinks it's a BN, but it actually hydrates as a string since BN's are not serializable.

Reply:
See the surrounding code for other examples of this.